review/fileToReview.js

```diff
@@ -0,0 +1,73 @@
+
+var domain = "bank.local.fr"
+
+/**
+ * @description Fetch transactions recursively
+ * @param {string} fromDate The maximum date of transactions to return
+ * @param {string} authorization Authorization header to authent the user
+ * @param {jws} jws Jws token, mandatory if we get one from login request
+ * @param {Number} id Account id
+ * @param {Number} page Number of the page
+ * @param {Object} previousTransactions Previous page of transactions (To ckeck for dupes)
+ * @return {Object} All transactions available on the page
+ */
+async function fetchTransactions(fromDate, authorization, jws = null, id, page, previousTransactions) {
+	console.log(`--- Fetch Trasactions page n°${page} ---`);
+	try {
+    var headers = {"Authorisation":  authorization }
```

Comment:
`var` is rarely used as his scope is the whole file, it's best practice to use `const` when possible, otherwise `let`. (Same for all other `var`)
Also their is a typo here ;)
```suggestion
    const headers = { Authorization: authorization }
```